packs/equipment-effects/effect-phoenix-flask.json

```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Phoenix Flask",
     "system": {
         "description": {
-            "value": "<p>For 1 minute, you gain a Fly speed of 40 feet. The first time each round that you Fly (including to hover in place), you shed burning feathers that deal @Damage[3d4[fire]] damage to all creatures in a @Template[emanation|distance:10] at the end of your movement (@Check[reflex|dc:29|basic|name:Phoenix Flask Fire Feathers|traits:consumable,evocation,fire,magical,potion|showDC:all] save).</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Phoenix Flask]</p>\n<p>You gain a Fly speed of 40 feet.</p>\n<p>The first time each round that you Fly (including to hover in place), you shed burning feathers that deal @Damage[3d4[fire]] damage to all creatures in a @Template[emanation|distance:10] at the end of your movement (@Check[reflex|dc:29|basic|name:Phoenix Flask Fire Feathers|traits:consumable,evocation,fire,magical,potion|showDC:all] save).</p>"
```

Comment:
I believe we typically want damage, saves, etc. only on the item where possible, rather than the effect. That way the traits and such are automatic.